src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Compilation.targets

```diff
@@ -104,7 +110,14 @@ Copyright (c) .NET Foundation. All rights reserved.
       <UseSharedCompilation>true</UseSharedCompilation>
     </PropertyGroup>
 
-    <Csc
+    <PropertyGroup>
+      <LangVersion Condition="'$(LangVersion)' == '' AND
```

Comment:
Do me a favour, do a search, do we have one use of `<Csc` or two

Reply:
😢  2

https://github.com/aspnet/AspNetCore-Tooling/blob/cfee40a19eb42f0c02365104c0a460e1025ad84c/src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Component.targets

https://github.com/aspnet/AspNetCore-Tooling/blob/99938c9b7e62896060b7d6f8a225cceb28da2c8d/src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Compilation.targets

I assume we want to maintain both of these right?

Reply:
@pranavkm @rynowak ping